network/jbuild

```diff
@@ -32,7 +32,7 @@ let () = Printf.ksprintf Jbuild_plugin.V1.send {|
  ((name xcp_network_interface)
   (public_name xcp.network.interface)
   (modules (network_interface))
-  (flags (:standard -w -39 %s))
+  (flags (:standard -w -39 -w -27 %s))
```

Comment:
Remove `-w -27`. If you rebase over `xcp-idl` master we merged changes to the `Makefile` to avoid failing in case of warning